easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,430 @@
+    }),
+    ('blist', '1.3.6', {
+        'checksums': ['3a12c450b001bdf895b30ae818d4d6d3f1552096b8c995f0fe0c74bef04d1fc3'],
+    }),
+    ('paycheck', '1.0.2', {
+        'checksums': ['6db7fc367c146cd59d2327ad4d2d6b0a24bc1be2d6953bb0773cbf702ee1ed34'],
+    }),
+    ('pbr', '5.6.0', {
+        'checksums': ['42df03e7797b796625b1029c0400279c7c34fd7df24a7d7818a1abb5b38710dd'],
+    }),
+    ('Cython', '0.29.24', {
+        'checksums': ['cdf04d07c3600860e8c2ebaad4e8f52ac3feb212453c1764a49ac08c827e8443'],
+    }),
+    ('six', '1.16.0', {
+        'checksums': ['1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926'],
+    }),
+    ('toml', '0.10.2', {
+        'checksums': ['b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f'],
+    }),
+    ('setuptools_scm', '5.0.2', {
+        'checksums': ['83a0cedd3449e3946307811a4c7b9d89c4b5fd464a2fb5eeccd0a5bb158ae5c8'],
+    }),
+    ('python-dateutil', '2.8.2', {
+        'modulename': 'dateutil',
+        'checksums': ['0123cacc1627ae19ddf3c27a5de5bd67ee4586fbdd6440d9748f8abb483d3e86'],
+    }),
+    ('decorator', '4.4.2', {
+        'checksums': ['e3a62f0520172440ca0dcc823749319382e377f37f140a0b99ef45fecb84bfe7'],
+    }),
+    ('liac-arff', '2.5.0', {
+        'modulename': 'arff',
+        'checksums': ['3220d0af6487c5aa71b47579be7ad1d94f3849ff1e224af3bf05ad49a0b5c4da'],
+    }),
+    ('pycrypto', '2.6.1', {
+        'modulename': 'Crypto',
+        'patches': ['pycrypto-2.6.1_remove-usr-include.patch'],
+        'checksums': [
+            'f2ce1e989b272cfcb677616763e0a2e7ec659effa67a88aa92b3a65528f60a3c',  # pycrypto-2.6.1.tar.gz
+            # pycrypto-2.6.1_remove-usr-include.patch
+            '06c3d3bb290305e1360a023ea03f9281116c230de62382e6be9474996086712e',
+        ],
+    }),
+    ('ecdsa', '0.17.0', {
+        'checksums': ['b9f500bb439e4153d0330610f5d26baaf18d17b8ced1bc54410d189385ea68aa'],
+    }),
+    ('enum34', '1.1.10', {
+        'modulename': 'enum',
+        'checksums': ['cce6a7477ed816bd2542d03d53db9f0db935dd013b70f336a95c73979289f248'],
+    }),
+    ('argparse', '1.4.0', {
+        'checksums': ['62b089a55be1d8949cd2bc7e0df0bddb9e028faefc8c32038cc84862aefdd6e4'],
+    }),
+    ('ipaddress', '1.0.23', {
+        'checksums': ['b7f8e0369580bb4a24d5ba1d7cc29660a4a6987763faf1d8a8046830e020e7e2'],
+    }),
+    ('asn1crypto', '1.4.0', {
+        'checksums': ['f4f6e119474e58e04a2b1af817eb585b4fd72bdd89b998624712b5c99be7641c'],
+    }),
+    ('idna', '2.10', {
+        'checksums': ['b307872f855b18632ce0c21c5e45be78c0ea7ae4c15c828c20788b26921eb3f6'],
+    }),
+    ('pycparser', '2.20', {
+        'checksums': ['2d475327684562c3a96cc71adf7dc8c4f0565175cf86b6d7a404ff4c771f15f0'],
+    }),
+    ('cffi', '1.14.6', {
+        'checksums': ['c9a875ce9d7fe32887784274dd533c57909b7b1dcadcc128a2ac21331a9765dd'],
+    }),
+    ('cryptography', '3.3.2', {
+        'preinstallopts': "export CARGO_HOME=%(builddir)s/cargo && ",
```

Comment:
This line shouldn't be here, that only makes sense for cryptography versions 3.4.x and newer (which require Rust as a build dependency)

Reply:
removed in f55296948b